server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/utils/JsonUtils.java

```diff
@@ -49,4 +61,149 @@ public static String extractValueFromJsonString(String inId, String inJson) {
             return outList;
         }
     }
+
+    public static JSONObject createUserFinalQuotaJson(String username, IUserFinalQuotaRepository finalQuotaRepository) throws ParseException{
+        UserFinalQuota quota = finalQuotaRepository.getQuotaByUsername(username);   // get default allocation
+        JSONObject outJson = new JSONObject();
+        if (quota != null) {
+            // get user's limit
+            UserUsages limit = new UserUsages();
```

Comment:
new UserUsages() initialization is not needed

Reply:
no it is needed in here. If you removed it, you will see the erros.

Reply:
I condensed it different way.

Reply:
condensed it.